set.mm

```diff
@@ -345636,15 +346052,167 @@ the identity of betweenness (Axiom A6).  (Contributed by Thierry
+        XAXBZUUDALMKALUWSWFZUXJUXLUUGZUUHUUIAUWRUWKUWMVHVKVCZVQVRZAUWKUWMUYQVQA
+        UWIUWLUWJUXAUXGUXDAUWIMUWLVQAUWIMVQVRLMKVMVCVQVRUOALMKUWSUXCAKUWTAKUWTU
+        YNULXCXDUUEXMAMUXJXNXOUUFAUWMAUWLUWJAMVHUXJUXKXPAMKUXJUXLXPWJXNXOAUWKVN
+        VDUYHUWMVNVDVGUWMXEVRUUJUWRUYRXJUXEUXFAUWMUYIUUKUWKVHUWMUULXBXMVGVHUWNW
+        AUUMUUNUUOAUYJMVMVCZKUXHVMVCZVKVCZUWDGVCZUYSUYTUWDGVCZGVCZUWPUWBAIXQVDZ
+        UYSFVDZUYTFVDZUWDDVDZVUBVUDVFAIUHUUQZAIUUPVDZUYJFVDZMFVDZMVGXFVUFUHAVUE
+        LFVDZVULVUKVUIAUWGFLUFUPWCZAUXBFMAUXBUWGFAVGUWGVDLUWGVDUXBUWGVTUURUPVGV
+        HVGLUUSWIUFUUTURWCZIUVAXRZFILMVUPYBZUEXSXBZVUOUKUYJMVUPFIVUQUEXTYCAVUJK
+        FVDZUXHFVDZUXHVGXFVUGUHAUWGFKUFUQWCZAVUEVHFVDZVUSVUTVUIAUWGFVHUFVHUWGVD
+        AUVBWSWCZVVAVUPFIVHKVUQUEXSXBZUXPKUXHVUPFIVUQUEXTYCAIUVCVDZQDVDZPDVDZVU
+        HAVUEVVEVUIIUVDYDZUDUCDINQPTUAYEXBZUYSUYTGVUPFDIUWDTVUQUBUEYFYCAVUAUWNU
+        WDGAVUAUYKUXIVMVCUWNAUYJMKUXHAUYJUYLWFZUXJUXLUXNUKUXPUVEAUYKUWKUXIUWMVM
+        UYPUXMYGYKYHAMUYJGVUPFDIVUCUWBTUBVUQUEUHVUOVURAVUCPBNVCZDAUXHKGVUPFDIUW
+        DVVKTUBVUQUEUHVVDVVAVVIAVVEVVGBDVDZVVKDVDVVHUCUIDINPBTUAYEXBZUXPULAKUXH
+        GVUPFDIUWDVVKTUBVUQUEUHVVAVVDVVIVVMULAUXHVVKGVCUXHKQBNVCZGVCZGVCZKUXHVV
+        NGVCZGVCZKUWDGVCAVVKVVOUXHGUSYIAKUXHVKVCZVVNGVCZUXHKVKVCZVVNGVCZVVRVVPA
+        VVSVWAVVNGAKUXHUXLUXNYJYHAVUEVUSVUTVVNDVDZVVTVVRVFVUIVVAVVDAVVEVVFVVLVW
+        CVVHUDUIDINQBTUAYEXBZKUXHGVUPFDIVVNTVUQUBUEYFYCAVUEVUTVUSVWCVWBVVPVFVUI
+        VVDVVAVWDUXHKGVUPFDIVVNTVUQUBUEYFYCYLAVVQUWDKGAVVQVVNVVKNVCZUWDAVHKVUPU
+        VGXRZVCZVVNGVCVHVVNGVCZVVONVCVVQVWEAVHKVWFGVUPFNDIVVNTUBVUQUEUAVWFYBZAV
+        UEIUVHVDZVUIIUVFYDZVVCVVAVWDYMAUXHVWGVVNGAVUEVVBVUSUXHVWGVFVUIVVCVVAVHK
+        VUPFIVUQUEYNXBYHAVVNVWHVVKVVONAVWHVVNAVUEVWCVWHVVNVFVUIVWDGDIVVNTUBUVIX
+        LUVJUSYGYOAVVEVVFVVGVVLVWEUWDVFVVHUDUCUIDINQPBTUAYSYCYKYIYPYQYRZVVMYTAV
+        VECDVDZVVGUWBDVDVVHACBLOBNVCZGVCZEVCZDVAAVVEVVLVWODVDZVWPDVDVVHUIAVWJVU
+        MVWNDVDZVWQVWKVUNAVVEODVDVVLVWRVVHUJUIDINOBTUAYEXBZLGVUPFDIVWNTVUQUBUEU
+        VKXBZDEIBVWOTUGUVLXBYTZUCDINCPTUAYEXBZUKALMUYOUXJUNWLZAVUCVVKMUYJVMVCUW
+        BGVCVWLAUYJMGVUPFDIUWBVVKTUBVUQUEUHVURVUOVXBVVMVXCUKAMUYJGVUPFDIUWBVVKT
+        UBVUQUEUHVUOVURVXBVVMUKAUYJVVKGVCUYJMVWNGVCZGVCZMUYJVWNGVCZGVCZMUWBGVCA
+        VVKVXDUYJGUTYIAMUYJVKVCZVWNGVCZUYJMVKVCZVWNGVCZVXGVXEAVXHVXJVWNGAMUYJUX
+        JVVJYJYHAVUEVULVUKVWRVXIVXGVFVUIVUOVURVWSMUYJGVUPFDIVWNTVUQUBUEYFYCAVUE
+        VUKVULVWRVXKVXEVFVUIVURVUOVWSUYJMGVUPFDIVWNTVUQUBUEYFYCYLAVXFUWBMGAVXFC
+        BNVCZVVKNVCZUWBALMVWFVCZVWNGVCVWOVXDNVCVXFVXMALMVWFGVUPFNDIVWNTUBVUQUEU
+        AVWIVWKVUNVUOVWSYMAUYJVXNVWNGAVUEVUMVULUYJVXNVFVUIVUNVUOLMVUPFIVUQUEYNX
+        BYHAVXLVWOVVKVXDNAVXLVWPBNVCZVWOACVWPBNVAYHAIUVMVDZVVLVWQVXOVWOVFAVWJVX
+        PVWKIUVNYDUIVWTDEINBVWOTUGUAUVOXBYKUTYGYOAVVEVWMVVGVVLVXMUWBVFVVHVXAUCU
+        IDINCPBTUAYSYCYKYIYPYQYRUVQYRUVPUWFUWQVBUWNUWGUWCUWNVFUWEUWPUWBUWCUWNUW
+        DGUVRUVSUVTXLADGVBHIJNXQPQCRSTUAUBUCUDVUIVXAUWAXM $.
     $}
 
 $(
+    @{
```

Comment:
Why is this commented out?

Reply:
Because I still need to work on that proof. Corner cases are not handled, and one of the conditions to apply the lemma still evades me.